vkdgen.py

```diff
@@ -47,47 +47,53 @@
 alias int32_t = int;
 alias int64_t = long;
 
-@nogc pure nothrow {
-	uint VK_MAKE_VERSION(uint major, uint minor, uint patch) {
+@nogc pure nothrow {{

```

Comment:
> I don't understand this.

The commit message is "Changed flow back into original khronos structure". This has nothing to do with changing how the templates get populated.

And I don't care if `format` is more "modern" if it introduces a huge amount of boilerplate in the form of escaping `{` and `}`, which is extremely common in D. `%` and `replace` are not going anywhere soon (though honestly I should just get rid of the `NAMEPREFIX` stuff).